assets/src/components/v2/pre_fare_single_screen_alert.tsx

```diff
@@ -0,0 +1,288 @@
+import useTextResizer from "Hooks/v2/use_text_resizer";
+import React from "react";
+
+import { classWithModifier, classWithModifiers, formatCause, imagePath } from "Util/util";
+
+interface PreFareSingleScreenAlertProps {
+  issue: string;
+  location: string | null;
+  cause: string;
+  remedy: string;
+  routes: string[];
+  unaffected_routes: string[];
+  effect: string;
+  region: string;
+  updated_at: string;
+}
+
+const doFallbackLayout = (issue: string | null, remedy: string | null, pioSecondaryTextSize: string) => ( 
+  <>
+    <img
+      className="alert-card__pio-text__icon"
+      src={imagePath("clock-negative.svg")}
+    />
+    {issue && <div className="alert-card__pio-text__main-text">{issue}</div>}
+    {remedy && <div className={classWithModifier("alert-card__pio-text__secondary-text", pioSecondaryTextSize)}>{remedy}</div>}
+  </>
+)
+
+const standardIssueSection = (issue: string, location: string | null, textSize: string) => (
+  <div className="alert-card__issue">
+    <img
+      className="alert-card__icon"
+      src={imagePath("no-service-black.svg")}
+    />
+    <div>
+      <div className={classWithModifier("alert-card__content-block__text", textSize)}>
+        {issue}
+      </div>
+      {location && <div className="alert-card__issue__location">{location}</div>}
+    </div>
+  </div>
+)
+
+const downstreamIssueSection = (issue: string) => (
+  <div className="alert-card__issue">
+    <div className={classWithModifier("alert-card__content-block__text", "medium")}>{issue}</div>
```

Comment:
Hmmmmmmmmm, good question. I don't think just using a string would be my favorite (I'm not sure how I'd do it.) So there's [this option](https://mbta.slack.com/archives/D02SP50JKU7/p1686662076427279), or FreeText

Reply:
I decided to go with adding an endpoints list. It'll be an optional field that will only exist on shuttles and suspensions. It just uses the endpoint logic that already existed for other alert types. It is either `nil` or is a list of two elements. If it is an alert that only affects a single station, the list will still have two items but each item is the same.

Reply:
A shuttle / suspension that only affects 1 station is not included in the design, I think because it shouldn't happen in real-life. BUT I still think it's good that you addressed in on the backend. But I'm going to build the frontend assuming it will always be a range, and at worst it could say "No trains between  Airport &  Airport", which is silly, but so is the alert, and it gets the point across.